tools/Dockerfile-tools

```diff
@@ -14,6 +14,6 @@ RUN apt-get update \
         peco \
         vim \
         expect \
-    && curl --http1.1 -L https://cnfl.io/cli | sh -s -- -b /usr/bin \
+    && curl --http1.1 -L https://cnfl.io/cli | sh -s -- -b /usr/bin v1.18.0 \
```

Comment:
Is this the most recent version of the CLI with acceptable performance?

Reply:
I did not do an analysis to find the latest between `v1.18.0` and `v1.23.0` that has acceptable performance; I just went with the known working version.

Ideally this version pinning is short term, and when https://confluentinc.atlassian.net/browse/CLI-851 is resolved, this code can go back to `latest`